python/nano/src/bigdl/nano/deps/openvino/tf/model.py

```diff
@@ -39,11 +40,14 @@ def __init__(self, model):
             if isinstance(model, tf.keras.Model):
                 export(model, str(dir / 'tmp.xml'))
                 ov_model_path = dir / 'tmp.xml'
-            OpenVINOModel.__init__(self, ov_model_path)
+            self.ov_model = OpenVINOModel(ov_model_path)
             AcceleratedKerasModel.__init__(self, None)
 
+    def forward_step(self, *inputs):
+        return self.ov_model.forward_step(*inputs)
+
```

Comment:
```suggestion
            model = OpenVINOModel(ov_model_path)
            super().__init__(model)

```

Reply:
I think using `super` is better.

Reply:
And you don't have to override `forward_step`.

Reply:
Yes, `forward_step` in PyTorchOpenVINOModel can be stripped, but it seems that KerasOpenVINOModel need it because `AcceleratedKerasModel` don't have `forward_step` method.

Reply:
Or should we add `forward_step` in `AcceleratedKerasModel`?

Reply:
https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/utils/inference/tf/model.py#L39-L42 I think it has?

Reply:
seems it will call `forward_step` in `AcceleratedModel`, which directly raise an error.
https://github.com/intel-analytics/BigDL/blob/main/python/nano/src/bigdl/nano/utils/inference/model.py#L31-L32